CHANGELOG.md

```diff
@@ -25,6 +26,7 @@
 - Update python:3.11-slim Docker digest to 2a746e2 ([#2743](https://github.com/nf-core/tools/pull/2743))
 - Update actions/setup-python action to v5 ([#2739](https://github.com/nf-core/tools/pull/2739))
 - Update gitpod/workspace-base Docker digest to 45e7617 ([#2747](https://github.com/nf-core/tools/pull/2747))
+- Fix for #2754, disk space on gha runners ([#2755](https://github.com/nf-core/tools/pull/2755))
```

Comment:
@mashehu  - it'd be nice if the changelog bot didn't add to the changelog if it already has edits in the PR.. :) 

```suggestion
```

Reply:
it wouldn't have changed anything, if the added Changelog entry would have guessed the correct PR number 😉